openamp/overview.rst

```diff
@@ -65,13 +93,18 @@ There are some AMP fundamentals which influence the OpenAMP architecture.
 Topology
 ========
 
-The OpenAMP framework assumes a main controller to remote system architecture, but otherwise the **topology** of the different runtime systems may be star, chain or a combination.
+The OpenAMP framework assumes a main controller to remote system architecture, but otherwise the
+**topology** of the different runtime systems may be star, chain or a combination.
 
 .. image:: ../images/topo_types.svg
```

Comment:
The topo_types image is not rendered [here](https://openamp--54.org.readthedocs.build/en/54/openamp/overview.html#topology)

![image](https://github.com/user-attachments/assets/242667fe-b079-4f44-aef0-47552a927778)